src/org/exist/util/serializer/SAXToReceiver.java

```diff
@@ -34,9 +34,15 @@
 
     private Receiver receiver;
     private boolean inCDATASection = false;
+    private boolean sendWStoCharacters = false;
```

Comment:
@dizzzz on/off switch for ignorableWhitespace redirection to characters ... where is the problem?